dirty_cat/tests/test_gap_encoder.py

```diff
@@ -7,41 +7,72 @@
 from dirty_cat._utils import parse_version
 from dirty_cat.tests.utils import generate_data
 
+# Importing: 37, 130, 135, 175, 213-235, 331,
+#
+# 734->737,
+# 868, 888->896, 1038->1040
 
-def test_analyzer():
+#        with pytest.raises(
+#            ValueError,
+#            match=r"get_feature_names is deprecated"):
```

Comment:
If the code isn't used, it should be removed
```suggestion
```

Reply:
Oups, this was in the draft version.

Reply:
Will remove.